adhocteam/Head-Start-TTADP
# [TTAHUB-310] Fix erroneously named column header

## Description of change

One of the headers on the topic frequency graph's accessible data view was named incorrectly by... someone. I've renamed it to be correct.

## How to test
Visit the dashboard and view the topic/frequency bar graph. Note that the heading should say "Topic" and "Count."

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-310

## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
